### DISCUSSION TITLE
What's the recommanded solution for backing up Dokploy?

### DISCUSSION THREAD

**Discussion by matheoleger** [Dokploy/dokploy#477] [Q&A] [answered]
Hello,

I would like to know if there is a recommended solution for backing up the Dokploy configurations, services, applications, etc...?

I noticed that there is a database for Dokploy with all the configurations, but I can't back it up using the official Dokploy solution (with S3 destination, Backups, etc...).
I also noticed the `/etc/dokploy` folders with some configurations for the applications (but I assume that if I back up the database, that should be enough?)

What is the correct way to back up Dokploy? Should I use rclone/rsync "manually"?

(NB. I saw [this discussion](https://github.com/Dokploy/dokploy/discussions/369) to a similar question)

Thank you in advance.

**Comment by Siumauricio** [comment, marked as answer]
Currently we don't have a integrated option to backup the whole instance, maybe in the next updates I will work on this

This is the list of the things are important

1. The database of dokploy
2. The /etc/dokploy directory

**Reply by RDeluxe** [reply]
Hi! Thank you for your answer.

So theorically, if we were to reinstall Dokploy on a new machine we should only have to reimport the DB + overwrite the /etc/dokploy folder, and everything should work the way it was?

**Reply by rcauler** [reply]
Wouldn't it be necessary to back up the docker volumes in `/var/lib/docker/volumes` as well?

**Reply by Siumauricio** [reply]
@rcauler Correct, in case you have databases created by dokploy you have 2 options, backup `/var/lib/docker/volumes` or the other option is to use dokploy backups and then import them into the new dokploy server.
